src/controllers/transaction/TransactionDryRunController.ts

```diff
@@ -54,13 +57,24 @@ export default class TransactionDryRunController extends AbstractController<Tran
 		this.router.post(this.path, TransactionDryRunController.catchWrap(this.dryRunTransaction));
 	}
 
-	private dryRunTransaction: IPostRequestHandler<ITx> = async ({ body: { tx }, query: { at } }, res): Promise<void> => {
+	private dryRunTransaction: IPostRequestHandler<ITx> = async (
+		{ body: { tx, at, senderAddress } },
+		res,
+	): Promise<void> => {
 		if (!tx) {
 			throw new BadRequest('Missing field `tx` on request body.');
 		}
 
+		if (!at) {
```

Comment:
It doesn't seem like `at` needs to be required for this endpoint?

Reply:
I made it not required, required some changes in the service as well